### PR TITLE
Move docopy out of the installer

### PR DESCRIPTION
**Pull request by Itxaka**
It doesnt neccesary has to be attached to it and can be reused by other
components

Signed-off-by: Itxaka <igarcia@suse.com>

### CODE REVIEW

File: pkg/utils/copy.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	"fmt"
+	v1 "github.com/rancher-sandbox/elemental-cli/pkg/types/v1"
+	"github.com/zloylos/grsync"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+func DoCopy(config *v1.RunConfig) error {
```
Comment (mudler) [inline pkg/utils/copy.go:13]: the `DoCopy` name is pretty generic, but we are copying cOS here..

Just thinking out loud, maybe we can have a cOS struct so we can:

```
cos.Copy(dst)
cos.SetActive(..)
cos.SetPassive(..)
cos.SetRecovery(..)
``

**Review by mudler — APPROVED** [review]

File: pkg/utils/copy.go (hunk 1 above)
Comment (Itxaka) [reply pkg/utils/copy.go:13]: yes, I hate the name, but Cant come up with a decent naming for this.

CopyCos? I mean, in the original this is only done on the install, so there is no reusability, but there is also nothing that should tie this to the installer.

Maybe the cos struct is the way to go...

Comment (Itxaka) [reply pkg/utils/copy.go:13]: I think we can merge this, and see where it lead us, but Im thinking that yes, the cos struct may be the better way of linking this somewhere.

Comment (mudler) [reply pkg/utils/copy.go:13]: maybe we can just live with the cos utilities inside `cos.go` or a comment on the top so we don't get confused

**Pull request merged**